### PR TITLE
Chore/bump example run data

### PR DESCRIPTION
**Pull request by anthonycorletti**

### CODE REVIEW

File: .github/workflows/examples.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
   schedule:
     # At 05:00 UTC on Tuesdays
     - cron: "0 5 * * 2"
+  pull_request:
+    types: [opened, synchronize]
```
Comment (Ben-Epstein) [inline .github/workflows/examples.yaml]: 🧹

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -7,13 +7,16 @@ on:
   push:
     branches:
       - main
-  pull_request:
-    types: [opened, synchronize]
+  # pull_request:
+  #   types: [opened, synchronize]
```
Comment (Ben-Epstein) [inline .github/workflows/test.yaml]:
```suggestion
  pull_request:
    types: [opened, synchronize]
```

**Review by Ben-Epstein — APPROVED** [review]

File: populator_gpu.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+).pip_install_from_requirements(requirements_txt="requirements-modal.txt")
+stub = Stub(
+    name="populator",
+    image=image,
+)
+secret = Secret.from_name("examples-populator-secret")
+
+
+def _modalinclude(path: str) -> bool:
+    return path.endswith(".ipynb") and "checkpoint" not in path
```
Comment (setu4993) [inline populator_gpu.py:37]: Ha 😅.

**Review by setu4993 — APPROVED** [review]
Amazing to see this!

The comment in the image classification notebook (can't leave a comment in there) could use an update since we're selecting 100 labels instead of 10 earlier.

File: examples/image_classification/Image_Classification_using_PyTorch_and_🔭_Galileo.ipynb [hunk 4]
```diff
@@ -158,10 +158,10 @@
     "    assert label_col in col_names, f\"{label_col} is not an existing column\"\n",
```
Comment (setu4993) [inline examples/image_classification/Image_Classification_using_PyTorch_and_🔭_Galileo.ipynb:158]: <pre class="ql-syntax" spellcheck="false">Line #57.      data['train'] = data['train'].filter(lambda example: example[label_col] == 0).select(range(100))
</pre><p> </p><p>Was referring to the comment above this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rungalileo/examples/pull/38/discussion/#comment-1195694792'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Pull request merged**